//test error
//PASSES

int main() {
  return (fn ()->int {return 1;}) (0);
}